spec/integration/resource/api200/interconnect/update_spec.rb

```diff
@@ -1,13 +1,14 @@
 require 'spec_helper'
 
-RSpec.describe OneviewSDK::Interconnect, integration: true, type: UPDATE do
+klass = OneviewSDK::Interconnect
+RSpec.describe klass, integration: true, type: UPDATE do
   include_context 'integration context'
 
-  let(:interconnect) { OneviewSDK::Interconnect.find_by($client, {}).last }
+  let(:interconnect) { klass.find_by($client, name: 'Encl1, interconnect 1').first }
 
-  describe '#name_servers' do
-    it 'retrieves name servers' do
-      expect { interconnect.name_servers }.not_to raise_error
+  describe '#update' do
+    it 'self raises MethodUnavailable' do
+      expect { interconnect.update }.to raise_error(/The method #update is unavailable for this resource/)
```

Comment:
I would be good to have the error explicitly expected here, not only its message, like:

```ruby
expect { interconnect.update }.to raise_error(MethodUnavailable, /The method #update is unavailable for this resource/)
```